### DISCUSSION TITLE
First Alpha Release!

### DISCUSSION THREAD

**Discussion by tajmone** [tajmone/Sublime-PML#33] [News]
We're glad to announce the first official release of **Sublime PML** `v0.7.0` (Alpha).

After a long incubation period, the package has finally been updated to support the new PML 2.3.0 syntax and its breaking changes.

You can now install the package directly from its GitHub repository via Sublime Text's native Package manager (see [installation instructions]).

The package offers decent PML syntax support by implementing the most important nodes and capturing all other nodes as "generic nodes". Thanks to smart symbols indexing, Goto Definition functionality, completions and snippets, keyboard shortcuts and a dedicated color scheme, users can already use this package in production, even though it's still in its early stages.

Being in Alpha stage, the package is still experimental can be subject to frequent changes in its scoping conventions or any other features, if better approaches to handling the PML syntax are discovered.

Enjoy!

And don't forget ... You're feedback is most precious for the growth of this package.

[installation instructions]: https://github.com/tajmone/Sublime-PML/blob/main/README.md#installation-instructions "How to install Sublime PML from within Sublime Text"

**Comment by pml-lang** [comment]
This is great and very much appreciated!
